apps/aws-notebook/app.yaml

```diff
@@ -0,0 +1,46 @@
+---
+overview: |
+  # Jupter Notebook
+
+workspaces:
+- name: dev
+  checkouts:
+  - path: demo
+    repo:
+      git: https://github.com/pigfall/crafting-demo-solutions.git
+    version_spec: tzz/aws-ecs-connection-app
+    manifest:
+      overlays:
+      - file: apps/aws-notebook/manifest.yaml
+  env:
+  - AWS_CONFIG_FILE=/run/sandbox/fs/secrets/shared/aws-config
+  - OPENVPN_CONFIG_PATH=/run/sandbox/fs/secrets/shared/openvpn-config
+  - ECS_CLUSTER_NAME=
+  - SUBNET_ID=
+  - SERVICE_LAUNCH_TYPE=FARGATE
+  base_snapshot: base-notebook-v1
+resources:
+- name: aws-ecs-service
+  brief: AWS ECS Service
+  handlers:
+    on_create:
+      max_retries: 3
+      timeout: 1800s
+      use_workspace:
+        name: dev
+        run:
+          dir: demo/apps/aws-notebook/tf
+          cmd: |
+            set -ex
+            bash on_create.sh
```

Comment:
Please put a single command here:

```
cmd: ./on_create.sh
```

Before commit to this PR, use `chmod a+rx on_create.sh` to make sure the script is executable. And also put `set -ex` inside the script, if needed.